library/src/ui/index.js

```diff
@@ -85,4 +85,59 @@ window.CoNDeT.ui = {
       this.removeChild(childToRemove);
     },
   },
+
+  DisplayComponent: (function () {
+    function constructor() {}
+
+    constructor.prototype.setDeltaXY = function ({ x, y }) {
+      common.deltaXY = { x: x, y: y };
+    };
+
+    constructor.prototype.prepareData = function (tablesJSON) {
+      var tablesList = [];
+      for (let i = 0; i < tablesJSON.length; i++) {
+        var { name, classType, coordinates, columns, rows } = tablesJSON[i];
+        var rowsPrepared = [];
+        for (let j = 0; j < columns.conditions.length; j++) {
+          rowsPrepared.push(rows.conditions[i][1] || "");
+        }
+        for (let j = 0; j < columns.decisions.length; j++) {
+          rowsPrepared.push(rows.decisions[i][1] || "");
+        }
```

Comment:
`let` niby działa w IE, ale jak już zaczęliśmy dawać `var` to trzymajmy się konwencji  
Generalnie to już prawie zadziała. No prawie, bo `i` wskakuje na indeks tabeli. Trzeba dodać oddzielny licznik który mówi w którym indeksie `rows.conditions` jesteśmy i zwiększać tylko jak `rows.conditions[<ten indeks>][1] != null`. Analogicznie dla `rows.decisions`